ui/config/environment.js

```diff
@@ -25,10 +25,10 @@ module.exports = function(environment) {
 
     APP: {
       blockingQueries: false,
-      mirageScenario: 'topoSmall',
+      mirageScenario: 'smallCluster',
```

Comment:
This still has `blockingQueries` changed to `false` by default, just making sure that’s by choice 🤓

Reply:
It wasn't...but I also like it? Ha. Blocking queries + Mirage is kinda janky anyway.

Reply:
Oh shit, nvm, I'll flip it back. I forget that this impacts non-mirage too.